src/app/main/component/eco-news/components/news-preview-page/news-preview-page.component.spec.ts

```diff
@@ -0,0 +1,140 @@
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { CUSTOM_ELEMENTS_SCHEMA, NO_ERRORS_SCHEMA } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { RouterModule } from '@angular/router';
+import { TranslateModule } from '@ngx-translate/core';
+import { ACTION_CONFIG, ACTION_TOKEN } from '../create-edit-news/action.constants';
+import { NewsPreviewPageComponent } from './news-preview-page.component';
+import { FormControl, FormGroup, FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { CreateEcoNewsService } from '@eco-news-service/create-eco-news.service';
+import { of } from 'rxjs';
+import { NewsResponseDTO } from '@eco-news-models/create-news-interface';
+import { EcoNewsModel } from '@eco-news-models/eco-news-model';
+
+describe('NewsPreviewPageComponent', () => {
+  let component: NewsPreviewPageComponent;
+  let fixture: ComponentFixture<NewsPreviewPageComponent>;
+
+  const currentFormWithImageMock: FormGroup = new FormGroup({
```

Comment:
It's better to define all your testing data inside beforeEach function, to be able to get clear data in each test
The declaration should where it is now
Because for now, it's possible to face some dependencies between tests
Some data could be changed in one test and it may affect the other one

So it should be like this:
`let currentFormWithImageMock: FormGroup;`
`...`
`beforeEach(() => {`
`currentFormWithImageMock = new FormGroup(...);`
`...`
`}`

And the same for the other variables